apps/custom-config/app.js

```diff
@@ -2,266 +2,86 @@ var _ = require('@yoda/util')._
-    }
-    var action = queryObj.action
-    logger.info('the isFirstLoad is' + queryObj.isFirstLoad)
-    var isFirstLoad
-    logger.info('on Url----> query typeof queryObj.isFirstLoad:  ', typeof queryObj.isFirstLoad)
-    if (queryObj.isFirstLoad === 'false' || queryObj.isFirstLoad === false) {
-      isFirstLoad = false
-    } else if (queryObj.isFirstLoad === 'true' || queryObj.isFirstLoad === true) {
-      isFirstLoad = true
-    }
-
-    logger.info('on Url----> action: ' + action + '; urlObj.pathname: ' + urlObj.pathname + ';   isFirstLoad: ' + isFirstLoad)
-    if (urlObj.pathname === '/vt_words') {
-      this.oldTxt = queryObj.oldTxt
-      this.txt = queryObj.txt
-      this.py = queryObj.py
-      onVtWordSwitchStatusChanged(action, false)
-    } else if (urlObj.pathname === '/continuousDialog') {
-      onPickupSwitchStatusChanged(action, isFirstLoad)
-    } else if (urlObj.pathname === '/wakeupSoundEffects') {
-      onWakeupSwitchStatusChanged(action, isFirstLoad)
-    } else if (urlObj.pathname === '/standbyLight') {
-      onStandbyLightSwitchStatusChanged(action, isFirstLoad)
-    } else if (urlObj.pathname === '/firstLoad') {
-      var config = queryObj.config
-      onLoadCustomConfig(config)
-    }
-  })
-  function initStandbyLight () {
-    var switchValue = property.get('persist.sys.standbylightswitch')
-    if (switchValue === SWITCH_OPEN) {
-      activity.light.play(STANDBY_LIGHT_JS, {}, {shouldResume: true})
-    } else {
-      activity.light.stop(STANDBY_LIGHT_JS)
-    }
-  }
-
-  function onVtWordSwitchStatusChanged (action, isFirstLoad) {
-    if (action && !isFirstLoad) {
-      if (action === SWITCH_VT_UPDATE) {
-        activity.turen.deleteVtWord(this.oldTxt)
-        activity.turen.addVtWord(this.txt, this.py)
-        sendAddUpdateStatusToServer(action)
-        sendSuccessStatusToApp(action, true)
-      } else if (action === SWITCH_VT_ADD) {
-        activity.turen.addVtWord(this.txt, this.py)
-        sendAddUpdateStatusToServer(action)
-        sendSuccessStatusToApp(action, true)
-      } else if (action === SWITCH_VT_DELETE) {
-        activity.turen.deleteVtWord(this.txt)
-        sendDeleteStatusToServer(action)
-        sendSuccessStatusToApp(action, true)
-      }
-      activity.exit()
-    }
-  }
-
-  function onVtWordSwitchFirstChanged (action, isFirstLoad) {
-    if (action && !isFirstLoad) {
-      if (action === SWITCH_VT_UPDATE) {
-        activity.turen.deleteVtWord(this.oldTxt)
-        activity.turen.addVtWord(this.txt, this.py)
-      } else if (action === SWITCH_VT_ADD) {
-        activity.turen.addVtWord(this.txt, this.py)
-      } else if (action === SWITCH_VT_DELETE) {
-        activity.turen.deleteVtWord(this.txt)
-      }
-      activity.exit()
-    }
-  }
+var flora = require('@yoda/flora')
+var WakeupEffect = require('wakeupEffect')
+var StandbyLight = require('standbyLight')
+var ContinuousDialog = require('continuousDialog')
+var VtWord = require('vtWord')
```

Comment:
Ditto.

Reply:
And this file should be: wakeup-effect, standby-light, continuous-dialog, vt-word.